docs/reference/index-modules.asciidoc

```diff
@@ -115,8 +115,8 @@ Index mode supports the following values:
 
 `time_series`::: Index mode optimized for storage of metrics documented in <<tsds-index-settings,TSDS Settings>>.
 
-`logsdb`::: Index mode optimized for storage of logs. It applies default sort settings on the `hostname` and `timestamp` fields and uses <<synthetic-source,synthetic `_source`>>. <<index-modules-index-sorting,Index sorting>> on different fields is still allowed.
-preview:[]
+`logsdb`::: _(https://www.elastic.co/pricing[Enterprise subscription] required)_ +
+Index mode optimized for storage of logs. By default, this setting uses <<synthetic-source,synthetic `_source`>> and sorts on the `hostname` and `timestamp` fields. You can override the defaults to <<index-modules-index-sorting,sort>> on different fields.
```

Comment:
```suggestion
<<logs-data-stream,`logsdb`>>::: Index mode optimized for storage of logs. By default, this setting uses <<synthetic-source,synthetic `_source`>> and sorts on the `hostname` and `timestamp` fields. You can override the defaults to <<index-modules-index-sorting,sort>> on different fields.
```

Reply:
Might be helpful to link to main logs db page here? I don't think you need the subscription requirement here and can delegate to the overview page for that information.